homeassistant/components/sensibo/__init__.py

```diff
@@ -1 +1,62 @@
 """The sensibo component."""
+from __future__ import annotations
+
+import asyncio
+import logging
+
+import aiohttp
+import async_timeout
+import pysensibo
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import _INITIAL_FETCH_FIELDS, DOMAIN, PLATFORMS, TIMEOUT
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Sensibo from a config entry."""
+    client = pysensibo.SensiboClient(
+        entry.data[CONF_API_KEY], session=async_get_clientsession(hass), timeout=TIMEOUT
+    )
+    devicelist = []
```

Comment:
Please separate words with underscore in variable names. I'd just call this variable `devices`. Describing the type is often not needed in names and better done by type annotations.